frontend/public/components/catalog/catalog-item-details.jsx

```diff
@@ -72,7 +72,7 @@ export class CatalogTileDetails extends React.Component {
             <div className="modal-body-inner-shadow-covers">
               <div className="co-catalog-page__overlay-body">
                 <PropertiesSidePanel>
-                  <Link className="btn btn-primary co-catalog-page__overlay-create" to={href} role="button" title={this.props.item.createLabel} onClick={closeOverlay}>{this.props.item.createLabel}</Link>
+                  <Button className="co-catalog-page__overlay-create" component="a" href={href} role="button" title={this.props.item.createLabel} onClick={closeOverlay} variant="primary">{this.props.item.createLabel}</Button>
```

Comment:
This will cause the page to reload. We should almost never use `<Button component="a">` and need to use `Link`.